src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.block {
+  height: 300px;
+  width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
indicate font-family in html or body tag

Reply:
ok, changed